packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -118,30 +118,41 @@ export class AccountForm extends PureComponent {
     if (locales && lang) {
       extend(locales[lang])
     }
+
+    if (props.oauth) return
+    this.sanitizedFields = Manifest.sanitizeFields(props.fields)
```

Comment:
As we are in a Pure Component, we should not store value as inner properties. If the props change, we cannot be sure that we will keep sanitized fields.

Keeping both `sanitizedFields` and `defaultValues` computed in render method will ensure idempotency.

It will avoid the conditional instruction l.122